src/main/java/com/ceos22/cgv_clone/web/domain/User.java

```diff
@@ -0,0 +1,37 @@
+package com.ceos22.cgv_clone.web.domain;
+
+import com.ceos22.cgv_clone.global.common.BaseEntity;
+import jakarta.persistence.*;
+import lombok.*;
+
+import java.time.LocalDate;
+
+@Entity
+@Getter
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+public class User extends BaseEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "user_id")
+    private Long id;
+
+    @Column(nullable = false,length = 20)
+    private String name;
+
+    @Column(nullable = false, unique = true,length = 50)
+    private String email;
+
+    @Column(length = 50)
+    private String password;
+
+    @Column(length = 50,name = "phone_number")
+    private String phoneNumber;
+
+    @Column(name = "birth_date")
+    private LocalDate birthDate;
+
+    @Column(name = "is_deleted")
+    private boolean isDeleted;
```

Comment:
p2. Soft Delete 를 위한 필드로 보이는데, 실제 로직에는 적용되지 않은 것 같아요.
최소한 // TODO 로 미구현 상태라는걸 작성해두 면 좋을 것 같아요

```suggestion
    @Column(name = "is_deleted")
    private boolean isDeleted; // TODO: 탈퇴 유저 조회에서 제외하기
```